lib/generator.js

```diff
@@ -454,6 +454,37 @@ const generateContentsTable = (buf, levelLimit = 0) => {
     .map(([str, level]) => ' '.repeat(level * 2) + str);
 };
 
+const sortMethods = (methods, prioritizedEntries) => {
```

Comment:
Perhaps something like:
```js
const sortMethods = (methods, prioritizedEntries) =>
  common
    .iter(methods)
    .filter(([method]) => {
      const isPrioritized = prioritizedEntries.some(
        e =>
          method === e ||
          (method.startsWith(e) && method.slice(e.length)[0] === '.')
      );
      if (isPrioritized) methods.delete(method);
      return isPrioritized;
    })
    .chainApply(it =>
      it.toArray().sort(([a], [b]) => {
        if (a === b) return 0;
        let aIdx = prioritizedEntries.indexOf(a);
        let bIdx = prioritizedEntries.indexOf(b);
        if (aIdx === -1) aIdx = Infinity;
        if (bIdx === -1) bIdx = Infinity;
        if (aIdx < bIdx) return -1;
        if (aIdx > bIdx) return 1;
        if (a < b) return -1;
        if (a > b) return 1;
        return 0;
      })
    )
    .chain(methods)
    .collectTo(Map);
```
Though this uses unreleased `chainApply`, but it can be replaced with an additional variable.

Reply:
`methods` shouldn't be changed since it is used later.